material-overrides/assets/stylesheets/polkadot.css

```diff
@@ -1369,7 +1379,27 @@ details.interface > summary::before {
 }
 
 .md-feedback__note {
-  font-size: 14px;
+  font-size: 12px;
+}
+
+.feedback-actions-container {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  align-items: center;
+  border: 1px solid var(--mystic);
+  border-radius: 16px;
+  padding: 1.5em;
+  width: fit-content;
+  background-color: var(--white);
+  margin-top: 2rem;
+}
+
+.feedback-actions-container .divider {
+  width: 100%;
+  height: 1px;
+  background-color: var(--mystic);
+  margin: 10px 10px 10px 10px;
```

Comment:
this could be cleaned up to `margin: 10px;` - not sure if this will get impacted by the general changes that need to happen with the spacing